Functions.Templates/Templates/EventHubTrigger-Python-v2-Append/eventhub_trigger_template.md

```diff
@@ -0,0 +1,35 @@
+# Azure Functions: Event Hub Trigger in Python
+
+## Event Hub Trigger
+
+The Event Hub function trigger can be used to respond to an event sent to an event hub event stream. You must have read access to the underlying event hub to set up the trigger. When the function is triggered, the message passed to the function is typed as a string.
+
+## Using the Template
+
+Following is an example code snippet for Event Hub Trigger using the [Python programming model V2](https://aka.ms/pythonprogrammingmodel) (currently in Preview).
+
+```python
+import logging
+import azure.functions as func
+app = func.FunctionApp()
+@app.function_name(name="EventHubTrigger1")
+@app.event_hub_message_trigger(arg_name="myhub", event_hub_name="samples-workitems",
+                               connection=""CONNECTION_SETTING"") 
```

Comment:
Check `""` issues. In all the places. Also in the above example, the variable was mentioned within `<` and `>` - please make the changes in all the places you use variables.